lib/LMSManagers/LMSFinanceManager.php

```diff
@@ -2014,6 +2011,10 @@ public function GetInvoiceList(array $params)
             $where .= ' AND d.closed = 0';
         }
 
+	if ($customer) {
+            $where .= ' AND d.customerid = ' . $customer;
```

Comment:
W tym miejscu masz gwarancję, że $customer jest integerem, a nie tekstem z podzapytaniem SQL?

Reply:
W pliku `invoiceslist.php` jest intval() zarówno dla `$_POST['customer']` i `$_GET['customer']` a tylko tam jest używana ta metoda więc nic by się nie zadziało w obecnym kodzie. Ale mimo to uzupełniłem kod metody w d880b26594f52cf1b732a09eb315ff0435d44eb2